pkg/reconciliation/reconcile_racks_test.go

```diff
@@ -1671,3 +1673,121 @@ func TestFailedStart(t *testing.T) {
 
 	mockClient.AssertExpectations(t)
 }
+
+func TestReconciliationContext_startAllNodes(t *testing.T) {
+
+	tests := []struct {
```

Comment:
A comment that briefly explains the fixture would be helpful :) I had to read through the test setup, which is rather complex, before understanding `racks`.